models/prices/base/prices_base_tokens.sql

```diff
@@ -28,5 +28,12 @@ FROM
     ('based-basedmarkets','base','BASED',0xba5e6fa2f33f3955f0cef50c63dcc84861eab663,18),
     ('usdc-usd-coin', 'base', 'USDC', 0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913, 6),
     ('wsteth-wrapped-liquid-staked-ether-20', 'base', 'wstETH',0xc1CBa3fCea344f92D9239c08C0568f6F2F0ee452,18),
-    ('aero-aerodrome-finance', 'base', 'AERO', 0x940181a94A35A4569E4529A3CDfB74e38FD98631, 18)
+    ('aero-aerodrome-finance', 'base', 'AERO', 0x940181a94A35A4569E4529A3CDfB74e38FD98631, 18),
+    ('degen-degen-base', 'base', 'DEGEN', 0x4ed4e862860bed51a9570b96d89af5e1b0efefed,18),
+    ('base-velocimeter', 'base', 'BVM', 0xd386a121991e51eab5e3433bf5b1cf4c8884b47a,18),
```

Comment:
Hi @0xOuterGod,
this token-id `base-velocimeter` is invalid.

Please verify.

Reply:
Verified, it's not available on Coinpaprika.